testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeWriteDatabricksCompatibility.java

```diff
@@ -367,7 +371,13 @@ private void testVacuumRemoveChangeDataFeedFiles(Consumer<String> vacuumExecutor
             // https://docs.delta.io/2.1.0/delta-change-data-feed.html#change-data-storage
             vacuumExecutor.accept(tableName);
 
-            Assertions.assertThat(s3.listObjectsV2(bucketName, changeDataPrefix).getObjectSummaries()).hasSize(0);
+            List<S3ObjectSummary> summaries = s3.listObjectsV2(bucketName, changeDataPrefix).getObjectSummaries();
+            Assertions.assertThat(summaries).hasSizeBetween(0, 1);
+            if (!summaries.isEmpty()) {
+                DatabricksVersion databricksRuntimeVersion = getDatabricksRuntimeVersion().orElseThrow();
+                Assertions.assertThat(databricksRuntimeVersion.isAtLeast(DATABRICKS_122_RUNTIME_VERSION)).isTrue();
+                Assertions.assertThat(summaries.get(0).getSize()).isEqualTo(0);
```

Comment:
What object is expected here?
document.
can you assert on its name as well?

Reply:
It's an empty `_change_data` directory. Added a comment and a name assertion.